readme.md

```diff
@@ -35,6 +35,7 @@ minify_html = true # to ensure correct rendering due to minification of whitespa
 [markdown]
 highlight_code = true # should be set to true unless the page has no code to highlight
 highlight_theme = "css" # this theme includes its own CSS-based styling of highlighting, so this should be set to CSS
+smart_punctuation = true # ... turns into ellipsis, quotes become curly, consecutive dashes turn into em/en dashes
```

Comment:
Instead of this, could we include something like 
```
# other Markdown options - as described in the Zola documentation - go here, and can be set according to user preference
```

Reply:
Done! Yes it is a standard Zola configuration and your suggestion makes more sense.